modules/depend_on_what_you_use/0.1.0/MODULE.bazel

```diff
@@ -0,0 +1,64 @@
+module(
+    name = "depend_on_what_you_use",
+    # Keep in sync with setup_step_2.bzl
+    bazel_compatibility = [">=5.4.0"],
+    compatibility_level = 0,
```

Comment:
We recommend leaving this at the default (i.e. not specifying it) as it isn't necessarily a good idea to bump this when you bump the major version. You can always set it explicitly later and its exact value doesn't matter.